tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/frontend/pages/EfilingPage.java

```diff
@@ -0,0 +1,30 @@
+package ca.bc.gov.open.jagefilingapi.qa.frontend.pages;
+
+import ca.bc.gov.open.jagefilingapi.qa.frontendutils.DriverClass;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class EfilingPage extends DriverClass {
+        private final WebDriver driver;
+        private final By cancelButton = By.xpath("//*[@id='root']/div/main/div/div/div/div/button");
+
+        //Initializing the driver
+        public EfilingPage(WebDriver driver) {
+            this.driver = driver;
+        }
+
+        //Actions:
+        public String verifyEfilingPageTitle() {
+            WebDriverWait wait = new WebDriverWait(driver, 30);
+            wait.until(ExpectedConditions.titleIs("eFiling Frontend"));
+            log.info("Waiting for the page to load...");
```

Comment:
Do we want to be more verbose about where this comment is coming from if we're going to be referring to logs in the future?

Reply:
Added a bit more detail to the logs. These logs are just to indicate that the particular step has been executed. Will configure to save it in a file with more details in the next update. Thanks!